examples/b2c/checkout/order-confirmation-email-flow/flows/B2B2C_Order_Confirmation_Email_Notification_Flow.flow

```diff
@@ -95,21 +107,23 @@
             <assignToReference>orderItems</assignToReference>
             <operator>Add</operator>
             <value>
-                <stringValue>&lt;/td&gt;&lt;td&gt;$</stringValue>
-            </value>
-        </assignmentItems>
-        <assignmentItems>
```

Comment:
Why is it removed?

Reply:
These are of total amount string values. To the UX https://www.figma.com/file/fbCXLQCt9PMk1aIrJxIXEe/246-Order-Confirmation-Email?type=design&node-id=783-23505&t=64Yey6WX1mt7ZwnV-0, I have removed it and kept only each product unit value.